assignments/this.js

```diff
@@ -13,14 +19,57 @@
 
 // code example for Window Binding
 
+function windowsBind(){
+      console.log(this)
+    }
+    
+    windowsBind();
+
 // Principle 2
 
 // code example for Implicit Binding
+const Features = {
+    color: 'red',
+    name: 'Adele',
```

Comment:
I'm rollin in the ddeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeep